tests/testutil.c

```diff
@@ -408,7 +408,7 @@ test_util_get_system_module_name (void)
     char * result;
 
     _test_util_system_module_name = malloc (64);
-    p = popen ("find /lib -name \"libc-*.so\" | head -1 | xargs basename"
+    p = popen ("find /lib -name \"libc[-.]*so*\" | head -1 | xargs basename"
```

Comment:
I just tested two different ubuntu64 machines with the original code. On one the x86_64 libc so was first, and on the other the i386 libc so was first. In other words, the original code wasn't perfect either...

Perhaps we should try something a little more precise?